src/prometheus/deploy/start.sh.template

```diff
@@ -25,6 +25,7 @@ kubectl create configmap prometheus-alert --from-file=alerting --dry-run -o yaml
 kubectl create configmap prometheus-record --from-file=recording --dry-run -o yaml | kubectl apply --overwrite=true -f - || exit $?
 kubectl apply --overwrite=true -f prometheus-configmap.yaml || exit $?
 kubectl apply --overwrite=true -f prometheus-deployment.yaml || exit $?
+kubectl apply --overwrite=true -f rbac.yaml || exit $?
```

Comment:
you need to create account and role binding before `prometheus-deployment.yaml`

Reply:
fixed

---
In reply to: [334220527](https://github.com/microsoft/pai/pull/3716#discussion_r334220527) [](ancestors = 334220527)